_notebooks/textbook.py

```diff
@@ -130,3 +144,22 @@ def clean_raw_text(self, disable_pysbd=False):
             if not disable_pysbd:
                 clean_text = "\n".join(seg.segment(clean_text))
             chapter.clean_text = clean_text
+    
+    def resolve_coreference(self):
+        """Uses spacy pipleline as a base and extends it
+        using neuralcoreference to process the coreferences
+        in the plain text that you get.
+
+        Saving both resolved text as well as the coreference
+        clusters in the chapter.
+        """
+        for chapter in tqdm(self.chapters):
+            if not chapter.clean_text :
```

Comment:
Do we have an alternative, more descriptive alternative for `clean_text`? Asking because in 6 months from now we will not remember what "clean" means in the overall text processing pipeline.